mount/mount_unix.go

```diff
@@ -71,7 +71,7 @@ func RecursiveUnmount(target string) error {
 		if err != nil {
 			if i == lastMount {
 				if suberr != nil {
-					return fmt.Errorf("%w (possible cause: %s)", err, suberr)
+					return fmt.Errorf("%w (possible cause: %w)", err, suberr)
```

Comment:
See my other comment; how much work would it be to continue making this work on older go versions? (I guess we could make the format string conditional)

Reply:
Or if we don't really need the `subErr` as an error, we could probably make the linter happy if we use `subErr.Error()`

Reply:
> Or if we don't really need the `subErr` as an error, we could probably make the linter happy if we use `subErr.Error()`

That was my initial patch exactly :)

When I figured we have a perfectly working version for older Go releases, and a new one may as well use Go 1.20.

Reply:
Done (with a TODO added to switch to `%w`).